pkgs/applications/version-management/git-sync/default.nix

```
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "git-sync";
-  version = "unstable-2022-03-20";
+  version = "unstable-2024-02-15";
```

```suggestion
  version = "0-unstable-2024-02-15";
```
This is the new recommended version scheme.

---

Thank you!